### PR TITLE
Removal of rake import

### PR DESCRIPTION
**Pull request by travega**
Removed the rails import rake task and specs.

### CODE REVIEW

File: lib/rails/generators/intercom/config/intercom.rb.erb [hunk 1]
```diff
@@ -21,8 +21,7 @@ IntercomRails.config do |config|
   <%- end -%>
 
   # == Intercom API Key
-  # This is required for some Intercom rake tasks like importing your users;
-  # you can generate one at https://app.intercom.io/apps/api_keys.
+  # You can generate one at https://app.intercom.io/apps/api_keys.
```
Comment (RuairiK) [inline lib/rails/generators/intercom/config/intercom.rb.erb]: I don't think we use REST API keys for anything other than the import task. We should get rid of this and all the underlying config unless there is another use for it that I'm not seeing.

Comment (travega) [inline lib/rails/generators/intercom/config/intercom.rb.erb]: Yeh I don't know. The description certainly sounds like they're not... So you reckon it's safe to remove?

Comment (RuairiK) [inline lib/rails/generators/intercom/config/intercom.rb.erb]: Pretty sure, can't see it being used anywhere else https://github.com/intercom/intercom-rails/search?utf8=%E2%9C%93&q=api_key&type=Code

Comment (travega) [inline lib/rails/generators/intercom/config/intercom.rb.erb]: 👍

**Comment by RuairiK** [comment]
I think this can go too https://github.com/intercom/intercom-rails/blob/master/lib/intercom-rails/exceptions.rb#L5

After that :shipit:

**Comment by lucasmazza** [comment]
@travega there is a `rake_tasks` block inside the Railtie that tries to load the rake task that was removed on this PR which breaks existing apps now.

**Comment by travega** [comment]
@lucasmazza Thanks for the heads up. Should be sorted now 👍

**Comment by paveldruzyak** [comment]
@travega v0.2.36 is still broken

**Comment by travega** [comment]
Hey @paveldruzyak could you describe a bit more about what you mean by it's still broken? The current version of the SDK is v0.3.4 and v0.2.36 is no longer supported as the V1 API was deprecated some months ago.

**Comment by paveldruzyak** [comment]
@travega same as described @lucasmazza: `rake_tasks` block inside the Railtie.

Interesting, we on v0.2.34. I didn't find changelog. Can we safely upgrade to latest version?

**Comment by travega** [comment]
@paveldruzyak these are the SDK releases: https://github.com/intercom/intercom-rails/releases. You'll see the notes for v0.3.0 is where the deprecated V1 support was removed. If you have any questions let me know 👍

**Comment by scottjacobsen** [comment]
Any plans to create a rake task like this for the latest intercom api? We used to use this.

**Comment by stevenharman** [comment]
With this case, should the `config.user.model` have also been removed and/or deprecated?

**Pull request merged**